src/controllers/breaches.js

```diff
@@ -20,7 +20,8 @@ async function breachesPage (req, res) {
     emailCount,
     partial: breaches,
     csrfToken: generateToken(res),
-    fxaProfile: req.user.fxa_profile_json
+    fxaProfile: req.user.fxa_profile_json,
+    nonce: res.locals.nonce
```

Comment:
Ugh, I see where the nonce gets nasty – having to pass it on every page.  If this bothers you, we could consider (under separate ticket) building a middleware or something that automatically passes a data object to main on every request, without having to explicitly add it.

On the other hand, sometimes it's good to be explicit 🤷